pkg/integrations/node_exporter/config.go

```diff
@@ -102,7 +102,8 @@ type Config struct {
 	CPUEnableCPUGuest                bool                `yaml:"enable_cpu_guest_seconds_metric,omitempty"`
 	CPUEnableCPUInfo                 bool                `yaml:"enable_cpu_info_metric,omitempty"`
 	CPUFlagsInclude                  string              `yaml:"cpu_flags_include,omitempty"`
-	DiskStatsIgnoredDevices          string              `yaml:"diskstats_ignored_devices,omitempty"`
+	DiskStatsDeviceExclude           string              `yaml:"diskstats_device_exclude,omitempty"`
+	DiskStatsDeviceInclude           string              `yaml:"diskstats_device_include,omitempty"`
```

Comment:
This would be considered a breaking change, so we'd need to document it as such in the upgrade guide.

Reply:
This shouldn't be *breaking* because there's a migration from the old to the new value. But I added a note to the changelog and upgrade-guide and also rebased on main to resolve the merge conflict.